src/commands/validate/threshold.ts

```diff
@@ -55,13 +56,23 @@ export default class Threshold extends Command {
     for (const statusThreshold of targets) {
       const [statusName, _total] = statusThreshold.split('.')
       if (_.get(thresholds, statusThreshold) !== undefined) {
-        exitNonZeroIfTrue(
-          Boolean(
-            _.get(overallStatusCounts, renameStatusName(statusName))              !==
-            _.get(thresholds, statusThreshold),
-          ),
-          `${statusThreshold}: ${_.get(overallStatusCounts, renameStatusName(statusName))} != ${_.get(thresholds, statusThreshold)}`,
-        )
+        if (flags.totalMinimum) {
+          exitNonZeroIfTrue(
+            Boolean(
+              _.get(overallStatusCounts, renameStatusName(statusName))              <
```

Comment:
shouldn't this actually be less than or equal to, not just strictly less than?

Reply:
No because if this is equal then it's passing

Reply:
whoops misread which was which for the comparison, so yeah you're right